templates/CRM/Sqltasks/Page/Templates.tpl

```diff
@@ -0,0 +1,308 @@
+<div id="sqltasks-templates">
+    <div class="help">
+        <p>{ts}Here you can manage configuration templates for SQL Tasks.{/ts}</p>
+        <p>{ts}The selected default template will be used whenever a user wants to create a new task.{/ts}</p>
+    </div>
+
+    <div class="crm-form-block">
+        <table>
```

Comment:
You can add `class="dataTable"` and remove the surrounding `<div>` to make this (mostly) match the look of other tables in Civi.